tests/integration/test_formatter_contract.py

```
@@ -0,0 +1,27 @@
+
+RUNTIME = "alchemmist/quality-graph@" + "a" * 40
+
+
+@pytest.mark.integration
+def test_prettier_can_format_fresh_project_without_staling_artifacts(tmp_path: Path) -> None:
+    project = Project.initialize(tmp_path, RUNTIME)
+    project.generate()
+    npx = which("npx")
+    assert npx is not None
+
+    subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)